vmicore/src/lib/os/windows/ActiveProcessesSupervisor.cpp

```diff
@@ -68,6 +76,19 @@ namespace VmiCore::Windows
         processInformation->memoryRegionExtractor = std::make_unique<VadTreeWin10>(
             kernelAccess, eprocessBase, processInformation->pid, processInformation->name, logging);
 
+        // Make sure that DTB values are page aligned since we rely on this behavior. If that ever changes we want to
```

Comment:
Please read Section 5.3.2 of "AMD64 Architecture Programmer’s Manual", Volume 2.
This code makes no sense at all.

Reply:
Good catch. This code was implemented in order to verify the assumption that PCID bits are not stored within the dtb value of _EPROCESS struct. However, this code was not tested on the correct server. PCID values are in fact stored in the dtb value of the _EPROCESS struct.

Reply:
@cakeless @K-Mayer I have removed PCID sanitization entirely. I am pretty sure it's not needed. We should verify this though.